evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -80,3 +84,48 @@ def pipeline_number(self):
     def batch_number(self):
         """Returns the number of batches which have been recommended so far."""
         return self._batch_number
+
+    def _create_ensemble(self):
+        next_batch = []
+
+        # Custom Stacked Pipelines
+        problem_type = list(self._best_pipeline_info.values())[0][
+            "pipeline"
+        ].problem_type
+        n_jobs_ensemble = 1 if self.text_in_ensembling else self.n_jobs
+        input_pipelines = []
+        for pipeline_dict in list(self._best_pipeline_info.values()):
+            pipeline = pipeline_dict["pipeline"]
+            pipeline_params = pipeline_dict["parameters"]
+            if hasattr(self, "_transform_parameters"):
+                parameters = self._transform_parameters(pipeline, pipeline_params)
+            if hasattr(self, "_selected_cols"):
+                if (
+                    "Select Columns Transformer"
+                    in pipeline.component_graph.component_instances
+                ):
+                    parameters.update(
```

Comment:
If I'm reading this correctly, this relies on `hasattr(self, "_transformer_parameters")` being True in order to create the `parameters` object being updated here and used in creating a new pipeline down below. Is there a situation where `self._transform_parameters` would not exist?

Reply:
Currently, we have `self._transform_parameters` in both the default and iterative algorithm. However, I put this check in since `self._transform_parameters` and `self._selected_cols` doesn't exist in the AutoMLAlgorithm abstract base class. I updated this to create an empty parameters dictionary if this attribute doesn't exist.